cmd/kube-scheduler/app/options/deprecated.go

```diff
@@ -120,20 +120,19 @@ func (o *DeprecatedOptions) ApplyTo(cfg *kubeschedulerconfig.KubeSchedulerConfig
 		}
 	}
 
-	// The following deprecated options affect the only existing profile that is
-	// added by default.
+	// Deprecated flags have an effect iff no config file was provided, in which
```

Comment:
clarify DeprecatedOptions#ApplyTo is only called iff no config file was provided... I had a hard time convincing myself this behavior was equivalent/safe until I realized that

Reply:
and maybe add that to the DeprecatedOptions#ApplyTo godoc

Reply:
Done in godoc